x-pack/plugins/observability_solution/infra/public/components/asset_details/tabs/processes/processes.tsx

```diff
@@ -109,13 +130,15 @@ export const Processes = () => {
     setSearchText('');
   }, [setUrlState]);
 
+  const isLoading = isPending(status);
+
   return (
-    <ProcessListContextProvider hostTerm={hostTerm} to={state.currentTimestamp}>
+    <ProcessListContextProvider hostTerm={hostTerm} to={toTimestamp}>
       <EuiFlexGroup direction="column" gutterSize="m" ref={ref}>
         <EuiFlexItem grow={false}>
           <SummaryTable
-            isLoading={loading && !response}
-            processSummary={(!error ? response?.summary : null) ?? { total: 0 }}
+            isLoading={isLoading}
+            processSummary={(!error ? data?.summary : null) ?? { total: 0 }}
```

Comment:
Can we simply this a bit? I got a bit confused with the inner conditions. WDYT?
```
const summary = data?.summary || { total: 0 }
```
```suggestion
            processSummary={(error ? { total: 0 } : summary)}
```

Reply:
absolutely! Thanks.